website/content/partials/api/restricted-endpoints.mdx

```diff
@@ -6,6 +6,7 @@
 
 API path                                    | Root | Admin
 ------------------------------------------- | ---- | -----
+`sys/activation-flags/.../activate`         | YES  | NO
```

Comment:
@schavis I changed this to be the full `sys/activation-flags/secrets-sync/activate` path. In the future if other activation-flags are added, they don't necessarily have to be restricted in the same way as secret sync's flag, so I should be more specific here. What do you think?

Second question - For Admin it is really "No, with a caveat" - On HCP Vault Dedicated it is enabled on the admin namespace. For Vault Enterprise it is not. Should we reflect this here somehow? I'm guessing not since these docs are geared towards self-hosted Vault